docs/src/pages/style/links/links.md

```diff
@@ -0,0 +1,25 @@
+---
+components: Link
+---
+
+# Links
+
+<p class="description">The Link component allows you to show links on the page, open them in a new window.</p>
```

Comment:
```suggestion
<p class="description">The Link component allows you to show a link on the page, and opens it in a new window.</p>
```

Reply:
By default, links don't open in a new window

Reply:
Yeah, my wording is crap.

Reply:
Then it probably shouldn't say that it does! (I was just correcting the grammar)...

Having said that, would a prop to do so optionally be a good idea? Looking at the benchmarks @oliviertassinari provided, this one is closest:

pinterest.github.io/gestalt/#/Link

It might also be helpful to explain when to use Link rather than `<a>`.